earthly/flutter/Earthfile

```diff
@@ -17,19 +17,19 @@ flutter-src:
     END
     SAVE ARTIFACT /flutter flutter
 
-# flutter-base - installs required tools and packages.
+# Installs required tools and packages.
 flutter-base:
     FROM debian:bookworm-20240513-slim
 
-    RUN apt-get update
-    RUN apt-get install -y git curl unzip bzip2 bash jq gpg lcov
+    RUN apt-get update --fix-missing
+    RUN apt-get install -y apt-utils git curl gzip unzip bzip2 bash jq gpg lcov
 
     WORKDIR frontend
 
-    ARG chrome_version = 126.0.6478.61
-    ARG chrome_package_release = 1
-    ARG chromium_version = 126.0.6478.126-1~deb12u1
-    DO installer+INSTALL_CHROME_LINUX64 --chrome_version=$chrome_version --chrome_package_release=$chrome_package_release --chromium_version=$chromium_version
+    ARG TARGET_ARCH = amd64
```

Comment:
Maybe use buildin args `TARGETARCH` 
https://docs.earthly.dev/docs/guides/multi-platform#making-use-of-builtin-platform-args